runtime/doc/news.txt

```diff
@@ -91,16 +85,6 @@ EVENTS
 
 • |vim.ui_attach()| callbacks for |ui-messages| `msg_show` events are executed in
   |api-fast| context.
-• New/enhanced arguments in these existing UI events:
-  • `cmdline_hide`: `abort` argument indicating if the cmdline was aborted.
-  • `cmdline_show`:
-    • Prompts that were previously emitted as `msg_show` events, are now routed
-      through `cmdline_show`.
```

Comment:
At least this pullet should stay under |news-breaking|. But entire section seems fine to stay for the extra notice; I think I also encounter GUIs that broke on added arguments to UI events (even though they should handle that gracefully).

Reply:
I've moved these back.